src/fides/api/ops/models/connectionconfig.py

```diff
@@ -48,6 +48,7 @@ class ConnectionType(enum.Enum):
     manual = "manual"  # Run as part of the traversal
     email = "email"
```

Comment:
I originally wanted to keep this in case we wanted to create generic email connectors in the future from the UI without having to create all the associated classes but we can bring this back if we need to later.

Reply:
Cool, I see you've removed "email"